src/utilities/AlertHelper.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Note from '../components/Note';
+import { SetFieldValues } from "../utilities/FormHelpers";
+export const getAlertMessage = (props) => {
+
+	const { hasPasswordReset } = props.values;
+	if (hasPasswordReset) {
+		return <Note className='bc_alert alert-success'>{props.values.SignInPage.ResetPasswordAlert.replace('{email address}', props.history.location.state)}</Note>
+	}
+	else if (props.status) 
+	{
+		const { incorrectEmail, networkError } = props.status;
+		const message = incorrectEmail || networkError || null;
+		return <Note icon='Nothing' className='bc_alert alert-warning'>{message}</Note>
+	}
+};
+
+export const AlertAtPage = (pageIn, props) => {
```

Comment:
As mentioned in the previous review, these functions should be moved to an `AlertHelpers` utility file. 

Component files shouldn't include functions that will be used for other components in cases like these.